lib/cylc/cfgspec/suite.py

```diff
@@ -309,34 +306,27 @@ def _coerce_parameter_list(value, keys, _):
             'env-script': vdr(vtype='string'),
             'err-script': vdr(vtype='string'),
             'pre-script': vdr(vtype='string'),
-            'script': vdr(
-                vtype='string',
-                default='echo Dummy task; sleep $(cylc rnd 1 16)'),
+            'script': vdr(vtype='string'),
```

Comment:
I was thinking just of the `script` with this comment. For a task like `[foo]script=true` the job.out now looks like this:

```
(no init-script defined)
Suite    : <suite-name>
Task Job : 1/<task-name>/01 (try 1)
User@Host: <user>@<host>

(no env-script defined)
(no pre-script defined)
2017-04-05T09:35:11+01 NORMAL - started
(no post-script defined)
2017-04-05T09:35:12+01 NORMAL - succeeded
```